extras/recsys/recsys.js

```diff
@@ -254,13 +263,6 @@ const recsys = {
     }
   },
 };
-// @if TARGET='web'
```

Comment:
Removing this because we think it won't catch anymore on close? This also triggers some entries when refocusing the tab, but don't think they are useful. Probably safe to remove...

Reply:
Ah, your right.  Let me know if we want to retain in the end, will revert.

Reply:
Probably retain for now, so Johnny doesn't think less calls are coming in.